### PR TITLE
test: Fix L0_dyna_implicit_state--base

### PR DESCRIPTION
**Pull request by krishung5**
#### What does the PR do?

The test erros out:
```
cp: cannot stat '/data/inferenceserver/24.11_a100_a100/qa_dyna_sequence_implicit_model_repository/*': No such file or directory 
```
Since `L0_dyna_implicit_state` test will `cd` into `L0_dyna_sequence_batcher` test directory, need to remove the `${REPO_VERSION}_${TEST_REPO_ARCH}` to fix the name of the test repo.

#### Checklist
- [ ] PR title reflects the change and is of format `<commit_type>: <Title>`
- [ ] Changes are described in the pull request.
- [ ] Related issues are referenced.
- [ ] Populated [github labels](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels) field
- [ ] Added [test plan](#test-plan) and verified test passes.
- [ ] Verified that the PR passes existing CI.
- [ ] Verified copyright is correct on all changed files.
- [ ] Added _succinct_ git squash message before merging [ref](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
- [ ] All template sections are filled out.
- [ ] Optional: Additional screenshots for behavior/output changes with before/after.

#### Commit Type:
Check the [conventional commit type](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type)
box here and add the label to the github PR.
- [ ] build
- [ ] ci
- [ ] docs
- [ ] feat
- [ ] fix
- [ ] perf
- [ ] refactor
- [ ] revert
- [ ] style
- [ ] test

#### Test plan:

- CI Pipeline ID: 20448566

#### Related Issues: (use one of the action keywords Closes / Fixes / Resolves / Relates to)
- closes GitHub issue: #xxx

### CODE REVIEW

File: qa/L0_dyna_implicit_state/test.sh [hunk 1]
```diff
@@ -34,9 +34,6 @@ if [ -z "$REPO_VERSION" ]; then
     echo -e "\n***\n*** Test Failed\n***"
     exit 1
 fi
-if [ ! -z "$TEST_REPO_ARCH" ]; then
-    REPO_VERSION=${REPO_VERSION}_${TEST_REPO_ARCH}
-fi
```
Comment (mc-nv) [inline qa/L0_dyna_implicit_state/test.sh:39]: may be it better to handle it in following way: 
```
if [ ! -z "$TEST_REPO_ARCH" ] && [[ "$REPO_VERSION" == *"$TEST_REPO_ARCH"* ]]; then
    REPO_VERSION=${REPO_VERSION}_${TEST_REPO_ARCH}
fi
```

File: qa/L0_dyna_implicit_state/test.sh [hunk 2]
```diff
@@ -34,7 +34,7 @@ if [ -z "$REPO_VERSION" ]; then
     echo -e "\n***\n*** Test Failed\n***"
     exit 1
 fi
-if [ ! -z "$TEST_REPO_ARCH" ]; then
+if [ ! -z "$TEST_REPO_ARCH" ] && [[ "$REPO_VERSION" == *"$TEST_REPO_ARCH"* ]]; then
```
Comment (mc-nv) [inline qa/L0_dyna_implicit_state/test.sh:37]: Suppose it's behavior should be:
```suggestion
if [ ! -z "$TEST_REPO_ARCH" ] && [[ "$REPO_VERSION" != *"$TEST_REPO_ARCH"* ]]; then
```

**Comment by krishung5** [comment]
@mc-nv Reran the test and it's passing with the latest changes. Please kindly review.

**Review by mc-nv — APPROVED** [review]

File: qa/L0_dyna_implicit_state/test.sh (hunk 2 above)
Comment (krishung5) [reply qa/L0_dyna_implicit_state/test.sh:37]: I think you're right, sorry I missed this. Let me fix it.

**Pull request merged**